dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -420,6 +421,22 @@ internal constructor(
             return this
         }
 
+
+        /**
+         * Enables/Disables tracking of frustration signals.
+         *
+         * By default frustration signals are tracked. Currently the SDK supports detecting
+         * error_taps which occur when an error follows a user action tap.
```

Comment:
```suggestion
         * error taps which occur when an error follows a user action tap.
```

Need also not to forget to update this statement when more signals are added.